### PR TITLE
finished backend for Google OAuth

### PR DESCRIPTION
**Pull request by michellekroll**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def googleAuth
```
Comment (ryan-garay89) [inline app/controllers/sessions_controller.rb]: This file is malformed (it needs a class definition) which is probably why the build is failing

**Review by ryan-garay89 — CHANGES_REQUESTED** [review]
Once the build is passing, please request review again

File: app/models/user.rb [hunk 2]
```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+    def self.from_omniauth(auth)
+        # Creates a new user only if it doesn't exist
+        where(email: auth.info.email).first_or_initialize do |user|
+          user.name = auth.info.name
+          user.email = auth.info.email
+        end
+      end
 end
```
Comment (KLongLo) [inline app/models/user.rb]: Also, would you mind changing the code to 2 spaces indent?

**Review by ryan-garay89 — APPROVED** [review]
LGTM!

**Pull request merged**